pkg/olm/olm.go

```diff
@@ -208,6 +208,10 @@ func RunCatalog(cmd *cobra.Command, args []string) {
 		util.Panic(os.WriteFile(dir+"noobaa-operator.package.yaml", pkgBytes, 0644))
 	}
 	util.Panic(util.WriteYamlFile(csvFileName, GenerateCSV(opConf, csvParams)))
+	// The CA configmap is needed prior to the operator startup to prevent a certificate injection race condition
+	util.Panic(util.WriteYamlFile(
```

Comment:
Probably needs to be only for ODF no? no ca inject outside of ODF as far as I know... 
The general question is what happens if we don't have anyone injecting the CAs to the operator? Does this mount delete the default file at /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem? the one coming from the OS?